src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -239,6 +239,17 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
+    def updateDateAccessed(googleId: UUID, dateAccessed: Instant): DBIO[Int] = {
+      clusterQuery.filter { _.googleId === googleId }.filter { _.dateAccessed < Timestamp.from(dateAccessed)}.map(_.dateAccessed).update(Timestamp.from(dateAccessed))
```

Comment:
Is this `filter` necessary? I would think that `updateDateAccessed` should update the column, regardless what the existing value is.